### DISCUSSION TITLE
Shouldn't we put the authentication before the `batch` creation?

### DISCUSSION THREAD

**Discussion by dadiorchen** [Greenstand/treetracker-earnings-api#31] [General]
https://github.com/Greenstand/treetracker-earnings-api/blob/e39b9e40d2725aa0b627cfb9071d4d140ccf4e3b/server/handlers/earningsHandler.js#L148-L155

**Comment by Kpoke** [comment]
@dadiorchen Yes, I think it should. This was implemented to get the admin username to store In the database, I am also not sure if there might be another way to get the username rather than sending the token since we do not really use it anywhere else in the API
